thirdparty/moab/MOABConfig.new.cmake.in

```diff
@@ -50,14 +50,8 @@ set(MOAB_HAS_FBIGEOM "@ENABLE_IGEOM@")
 set(MOAB_HAS_IMESH "@ENABLE_IMESH@")
 set(MOAB_HAS_MPI "@MOAB_HAS_MPI@")
 
-if(MOAB_USE_HDF)
-  if(EXISTS "@HDF5_DIR@/share/cmake/hdf5/hdf5-config.cmake")
-    include(@HDF5_DIR@/share/cmake/hdf5/hdf5-config.cmake)
-  endif()
-endif()
```

Comment:
I was just ripping stuff out that I didn't need.

I think the proper fix is to change it to
```
if(MOAB_USE_HDF)
  if(EXISTS "@HDF5_DIR@/hdf5-config.cmake")
    include(@HDF5_DIR@/hdf5-config.cmake)
  endif()
endif()
```
As at least for me HDF5_DIR points to ```<install_prefix>/share/cmake/hdf5/``` already.

I will update this branch to do exactly this.